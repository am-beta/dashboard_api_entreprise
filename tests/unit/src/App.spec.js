import App from "@/App.vue";
import { createLocalVue, shallowMount } from "@vue/test-utils";
import VueRouter from "vue-router";

const localVue = createLocalVue();
localVue.use(VueRouter);

describe("App.vue", () => {
  const wrapperApp = shallowMount(App, {
    localVue
  });

  test("It match the snapshot", () => {
    expect(wrapperApp.vm.$el).toMatchSnapshot();
  });
});
