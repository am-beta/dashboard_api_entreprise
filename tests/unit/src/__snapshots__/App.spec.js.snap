// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue It match the snapshot 1`] = `
<div
  id="app"
>
  <icon-definition-stub />
  <navigation-header-stub />
  <router-view-stub
    name="default"
  />
</div>
`;
